public/locales/en-US/grafana.json

```diff
@@ -1165,7 +1165,8 @@
           "new-recording-rule": "New recording rule",
           "title": "Grafana-managed"
         },
-        "loading-rules": "Loading rules from {{numberOfSources}} {{sources}}"
+        "loading-rules_one": "Loading rules from {{count}} sources",
```

Comment:
Oops - this should be "source"

Reply:
I don't think so 👀 https://github.com/grafana/grafana/blob/3de09fe4578b9724a72c469016a48df4d969445c/contribute/internationalization.md#L4